.github/workflows/pr-agent.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+  issue_comment:
+jobs:
+  pr_agent_job:
+    runs-on: ubuntu-latest
+    name: Run pr agent on every pull request, respond to user comments
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: Codium-ai/pr-agent@main
+        env:
+          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `pr-agent.yml` file seems to be correctly set up for the GitHub Actions workflow. However, it's important to ensure that the secrets `OPENAI_KEY` and `TOKEN_GITHUB` are properly set in the repository settings. Also, the action is triggered on both `pull_request` and `issue_comment` events, which might not be necessary if you only want to run this action on pull requests. If that's the case, consider removing the `issue_comment` trigger.

```diff
on:
  pull_request:
-3:   issue_comment:
jobs:
  pr_agent_job:
    runs-on: ubuntu-latest
    name: Run pr agent on every pull request, respond to user comments
    steps:
      - name: PR Agent action step
        id: pragent
        uses: Codium-ai/pr-agent@main
        env:
          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
\ No newline at end of file
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->